lesson_shell_fasta.md

```diff
@@ -0,0 +1,233 @@
+
+## Learning Objectives
+
+* Download files form FTP repositories
+* Working with compressed files
+* Working with FASTA files
+
+## Lesson
+
+In this lesson we will learn how to download data in [fasta format](https://en.wikipedia.org/wiki/FASTA_format) form the web and how to manipulate them using basic shell commands.
+We will use as sample data set the human [non-coding RNAs](https://en.wikipedia.org/wiki/Non-coding_RNA) (ncRNA).
+
+## Exercises
+### 1 - Downloading data
+
+The data we need are on the [Ensembl](http://www.ensembl.org/index.html) FTP site, a repository of all publicly available files from the project.
+
+There is an [html interface](http://www.ensembl.org/info/data/ftp/index.html) of the last release where it should be easy to locate the link to the FASTA file of the ncRNA
```

Comment:
Sarah, thanks for suggestions, I think they are great.
Would you edit the file or should I?
Enza


On 17 August 2015 at 16:07, Sarah Stevens <notifications@github.com> wrote:

> In lesson_shell_fasta.md
> <https://github.com/datacarpentry/shell-genomics/pull/7#discussion_r37190403>
> :
>
> > +
> > +* Download files form FTP repositories
> > +* Working with compressed files
> > +* Working with FASTA files
> > +
> > +## Lesson
> > +
> > +In this lesson we will learn how to download data in [fasta format](https://en.wikipedia.org/wiki/FASTA_format) form the web and how to manipulate them using basic shell commands.
> > +We will use as sample data set the human [non-coding RNAs](https://en.wikipedia.org/wiki/Non-coding_RNA) (ncRNA).
> > +
> > +## Exercises
> > +### 1 - Downloading data
> > +
> > +The data we need are on the [Ensembl](http://www.ensembl.org/index.html) FTP site, a repository of all publicly available files from the project.
> > +
> > +There is an [html interface](http://www.ensembl.org/info/data/ftp/index.html) of the last release where it should be easy to locate the link to the FASTA file of the ncRNA
>
> I'm hesitant to use the words *should be easy* because it implies some
> level of ability on the participants part. I would suggest just saying
> something like...
> In this interface, we need to locate the link to the FASTA file of human
> ncRNA on the page and click to download it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datacarpentry/shell-genomics/pull/7/files#r37190403>.
>



-- 

--------
https://bitly.com/vincenza
http://www.igb.cnr.it/obilab
http://www.bmtl.it/